Assets/Scripts/Editor/MissionCheats.cs

```diff
@@ -17,7 +17,7 @@ public static bool ValidateStartMissionTutorial()
     [MenuItem(Constants.CHEATS_STARTMISSIONTUTORIAL)]
     public static void StartMissionTutorial()
     {
-        MissionManager.Instance.StartMission(AssetDatabase.LoadAssetAtPath<GameObject>(Constants.ASSET_PATH_MISSIONTUTORIAL));
```

Comment:
can you remove the `Constants.ASSET_PATH_MISSIONTUTORIAL` constants now since we dont use them?